pkg/generators/forwarding/fluentd/fluent_conf.go

```diff
@@ -16,6 +18,45 @@ import (
 
 var replacer = strings.NewReplacer(" ", "_", "-", "_", ".", "_")
 
+type inputSelectorConf struct {
+	Pipeline        string
+	Namespaces      string
+	Labels          string
+}
+
+func newInputSelectorConf(pipeline string, namespaces []string, labelSelector *metav1.LabelSelector) (*inputSelectorConf, error) {
+	labelList := ""
+	namespaceList := ""
+
+	if namespaces != nil {
+		namespaceList = strings.Join(namespaces, ",")
+	}
+
+	labelMap, err := metav1.LabelSelectorAsMap(labelSelector)
+	if err != nil {
+		return nil, fmt.Errorf("LabelSelector: %v", err)
+        }
+	if labelMap != nil {
+                names := make([]string, 0, len(labelMap))
```

Comment:
Minor: append() works on a nil slice (creates a new slice) so you can do this;
```suggestion
                var names []string
```

Reply:
/lgtm